projects/airpuff/index.js

```diff
@@ -181,7 +207,12 @@ module.exports = {
       }
 
       const mswETH = {
-        valut: "0x7c505E03460aEF7FE88e218CC5fcEeCCcA4C4394",
+        vault: "0x7c505E03460aEF7FE88e218CC5fcEeCCcA4C4394",
+        reStakingToken: "0x32bd822d615A3658A68b6fDD30c2fcb2C996D678",
+      };
+
+      const mswETH1x = {
+        vault: "0x1100195fbdA2f22AA6f394E6C65f168779Fe572c",
```

Comment:
I dont think it is rpc issue, I dont see `getAllEigeinPieCycleDepositAmounts()` here: https://etherscan.io/address/0x1100195fbdA2f22AA6f394E6C65f168779Fe572c#readProxyContract